tests/components/coolmaster/test_config_flow.py

```diff
@@ -26,8 +24,7 @@ def _flow_data():
     assert result["errors"] is None
 
     with patch(
-        "homeassistant.components.coolmaster.config_flow.validate_connection",
-        return_value=mock_coro(True),
+        "pycoolmasternet.CoolMasterNet.devices", return_value=mock_coro([1])
```

Comment:
I think we need to patch the library name as imported in the config flow: `"homeassistant.components.coolmaster.config_flow.CoolMasterNet.devices"`.

Reply:
`CoolMasterNet.devices` is just a regular method, right? Ie, not a coroutine. So just set the `return_value` to the list of devices, `[1]`.